code/modules/clothing/under/marine_uniform.dm

```diff
@@ -705,15 +705,15 @@
 	sensor_faction = FACTION_CLF
 
 /obj/item/clothing/under/colonist/ua_civvies
-	name = "gray utilities"
+	name = "UA gray utilities"
 	desc = "A stylish gray jumpsuit - standard issue for UA civilian support personnel."
 	icon_state = "ua_civvies"
 	worn_state = "ua_civvies"
 	has_sensor = UNIFORM_HAS_SENSORS
 	sensor_faction = FACTION_MARINE
 
 /obj/item/clothing/under/colonist/wy_davisone
-	name = "brown utilities"
+	name = "UA brown utilities"
```

Comment:
add \improper before these (same with any non-proper names that begin with a capital letter) and turn it to "UA [color] utility uniform"

`\improper UA gray utilities` results in "That's a UA gray utility uniform." whereas without the `\improper` it just puts "That's UA gray utility uniform."

Reply:
once that's done I'll merge it....

Reply:
```suggestion
	name = "\improper UA brown utility uniform"
```